workflow-bot-app/src/test/java/com/symphony/bdk/workflow/utils/UtilityFunctionsMapperTest.java

```diff
@@ -0,0 +1,52 @@
+package com.symphony.bdk.workflow.utils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.symphony.bdk.workflow.engine.camunda.UtilityFunctionsMapper;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+@SuppressWarnings("unchecked")
+class UtilityFunctionsMapperTest {
+
+  @Test
+  void testString() {
+    final Object json = UtilityFunctionsMapper.json("This is a regular string");
+    assertThat(json).isEqualTo("This is a regular string");
+  }
+
+  @Test
+  void testInteger() {
+    final Object json = UtilityFunctionsMapper.json("2");
+    assertThat(json).isEqualTo(2);
+  }
+
+  @Test
+  void testBoolean() {
+    final Object json = UtilityFunctionsMapper.json("false");
```

Comment:
In that case will we receive true/false/2 (not strings) or strings?

Reply:
**"json instanceof String"** returns false, **"json instanceof Boolean"** returns true.
Does it worth to add it to the tests?

Reply:
I was expecting the test to be 
```     final Object json = UtilityFunctionsMapper.json(false);```
But I' mixing json() and the raw response again